bolt-modules/boltlib/spec/functions/run_task_spec.rb

```diff
@@ -356,9 +356,9 @@ def mock_task(executable, input_method)
 
       is_expected.to run.with_params(task_name, hostname, task_params).and_raise_error(
         Puppet::ParseError,
-        /Task\ test::params:\n
+        %r{Task\ test::params:\n
          \s*has\ no\ parameter\ named\ 'foo'\n
-         \s*has\ no\ parameter\ named\ 'bar'/x
+         \s*has\ no\ parameter\ named\ 'bar'}x
```

Comment:
These changes are just to make cli_spec a little more readable for VS Code users, as there's a bug with the Ruby plugin's syntax highlighting when you use multi-line regex literals.